crates/re_query/src/latest_at/results.rs

```diff
@@ -285,6 +285,5 @@ fn downcast<C: Component>(cached: &(dyn ErasedFlatVecDeque + Send + Sync)) -> cr
     if cached.num_entries() != 1 {
         return Err(anyhow::anyhow!("latest_at deque must be single entry").into());
     }
-    // unwrap checked just above ^^^
-    Ok(cached.iter().next().unwrap())
+    Ok(cached.iter().next().expect("checked just above ^^^"))
```

Comment:
if this would ever trigger then it's pretty hard to read on command line - "what is above" makes sense for the comment but not for the expect message.

```suggestion
    Ok(cached.iter().next().expect("checked existence of cached value already"))
```